app/models/users.py

```diff
@@ -0,0 +1,18 @@
+from typing import Union
+from sqlmodel import Field, SQLModel
+
+
+class Token(SQLModel):
+    access_token: str
+    token_type: str
+
+class TokenData(SQLModel):
+    username: Union[str, None] = None
+
+class Users(SQLModel, table=True):
+    username: str = Field(primary_key=True)
```

Comment:
It is fine to have the username as a primary key, as long as the username will never change.
I am unsure if this functionality is required.